tools/theme/src/Theme.ts

```diff
@@ -158,40 +175,56 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
 
     public override shadowRoot!: ShadowRootWithAdoptedStyleSheets;
 
-    private _theme: ThemeVariant | '' = 'spectrum';
-
+    private _system: SystemVariant | '' = 'spectrum';
     /**
-     * The Spectrum theme that is applied to the content scoped to this `sp-theme` element.
+     * The Spectrum system that is applied to the content scoped to this `sp-theme` element.
      *
      * A value is requried.
-     * @type {"spectrum" | "express" | ""}
+     * @type {"spectrum" | "express" }
      * @attr
      */
-    get theme(): ThemeVariant | '' {
-        const themeFragments = Theme.themeFragmentsByKind.get('theme');
+    get system(): SystemVariant | '' {
+        const themeFragments =
+            Theme.themeFragmentsByKind.get('system') ||
+            Theme.themeFragmentsByKind.get('theme');
```

Comment:
I'm not sure this fallback is correct. Why fallback on presence of theme fragments rather than on the presence of `this.system` vs `this.theme`?

Reply:
Only checking for `system` here now
```js
 get system(): SystemVariant | '' {
        const systemFragments =
            Theme.themeFragmentsByKind.get('system')
        .......
    }
 ```